src/batch-submitter/tx-batch-submitter.ts

```diff
@@ -380,12 +436,38 @@ export class TransactionBatchSubmitter extends BatchSubmitter {
     if (this.autoFixBatchOptions.fixDoublePlayedDeposits) {
       batch = await fixDoublePlayedDeposits(batch)
     }
-    if (this.autoFixBatchOptions.fixDelayedTimestampAndBlockNumberHardcoded) {
-      batch = await fixDelayedTimestampAndBlockNumberHardcoded(batch)
+    if (this.autoFixBatchOptions.fixMonotonicity) {
+      batch = await fixMonotonicity(batch)
     }
     return batch
   }
 
+  private async _getLastTimestampAndBlockNumber(): Promise<{ lastTimestamp: number, lastBlockNumber: number}> {
+    const addressManagerAddr = (await this._getRollupInfo()).addresses.addressResolver
```

Comment:
This is going to be removed in the new geth, we should pull the address manager address from a config option

Reply:
Ok sounds good